webrender_traits/Cargo.toml

```diff
@@ -13,6 +13,7 @@ webgl = ["offscreen_gl_context"]
 [dependencies]
 app_units = "0.4"
 byteorder = "1.0"
+core-foundation = "0.3"
```

Comment:
we could make it an OSX only dependency (via `
[target.'cfg(osx)'.dependencies]`)